hugegraph-server/hugegraph-dist/src/assembly/travis/test-check-port.sh

```
@@ -0,0 +1,984 @@
+    PARTIAL=""
+    curl() {
+        if [[ "$1" == "-fL" && "$2" == "-o" && \
+              "$4" == "--" && \
+              "$5" == "https://example.com/some/path/file.tar.gz" ]]; then
+            PARTIAL="$3"
+            # Simulate a partial/corrupted transfer: write something then fail.
+            echo "partial" > "$PARTIAL"
+            return 1
+        fi
+        return 1
+    }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)